### PR TITLE
Fix Checkbox not showing as reset after submit

### PR DESCRIPTION
**Pull request by rkuykendall**
Test failing if you remove Checkbox.tsx changes:
image

### CODE REVIEW

File: src/inputs/Checkbox.tsx [hunk 1]
```diff
@@ -2,20 +2,30 @@ import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 import autoBindMethods from 'class-autobind-decorator';
 import { omit } from 'lodash';
-import { ICheckboxProps } from '../interfaces';
 
 import * as Antd from 'antd';
 
-@autoBindMethods()
+import { IAntFormField, ICheckboxProps, IInjected } from '../interfaces';
+
```
[the remark this answers is not in the document]
Comment (rkuykendall) [reply src/inputs/Checkbox.tsx]: No, that's a good question, I was sorta on the fence. I left it for the same reason that I leave `autobindmethods` on a component even when I don't technically need it: We save a lot of developer time and sanity through consistency. See the other components that use `this.injected`:
image

[the remark this answers is not in the document]
Comment (ekatsuta) [reply src/inputs/Checkbox.tsx]: got it. thanks!

File: test/types/checkbox.test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { Tester } from '@mighty-justice/tester';
+
+import { TYPE_GENERATORS } from '../factories';
+
```
Comment (wsaffran) [inline test/types/checkbox.test.ts]: Is the space between these imports standard?

Comment (rkuykendall) [reply test/types/checkbox.test.ts]: Yeah, we usually put blank spaces between:
```
global imports
ant-design imports
@mighty-justice imports
../ imports
./ imports
style.module.less import
```

File: test/types/checkbox.test.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+describe('checkbox', () => {
+  it('resets on submit', async () => {
+    const onSave = jest.fn()
+      , props: Partial<IFormCardProps> = { fieldSets, onSave, resetOnSuccess: true };
+
+    const tester = await new Tester(FormCard, { props }).mount();
+
+    async function submitAndCheckFor (value: boolean) {
+      expect(onSave).not.toHaveBeenCalled();
+      expect(tester.find('.ant-checkbox-wrapper-checked').length).toBe(value ? 1 : 0);
```
Comment (SHA-4) [inline test/types/checkbox.test.ts]: You can use `exists()` to remove the ternary

**Pull request merged**